app/controllers/catalog_controller.rb

```diff
@@ -309,15 +309,12 @@ class CatalogController < ApplicationController
     config.add_show_tools_partial(:sms, if: :render_sms_action?, callback: :sms_action, validator: :validate_sms_params)
 
     # Custom tools for GeoBlacklight
-    config.add_show_tools_partial :metadata, if: proc { |_context, _config, options|
-      options[:document] && (Settings.METADATA_SHOWN & options[:document].references.refs.map { |r| r.type.to_s }).any?
-    }
-    config.add_show_tools_partial :arcgis, partial: 'arcgis', if: proc { |_context, _config, options|
-      options[:document] && options[:document].arcgis_urls.present?
-    }
-    config.add_show_tools_partial :data_dictionary, partial: 'data_dictionary', if: proc { |_context, _config, options|
-      options[:document] && options[:document].data_dictionary_download.present?
-    }
```

Comment:
these can now all come from geoblacklight components

Reply:
don't them need to be updated, not removed?

Reply:
to me the conditional logic for when to show the tools to me looks identical in geoblacklight, and i couldn't even find the partials that are referenced here in earthworks ... so I was presuming that since earthworks is derived from geoblacklight, they don't need to be re-defined here 

but i'm not 100% sure, would certainly be nice to validate with enough real data to be confident

Reply:
Checking core GeoBlacklight code, the ArcGIS and DataDictionary components are now included.  I also see the metadata partial being added similar to the above, so including the latest GeoBlacklight code on the main branch means we no longer require the additions above. For reference: https://github.com/geoblacklight/geoblacklight/blob/main/lib/generators/geoblacklight/templates/catalog_controller.rb#L309